packages/db/benchmarks/measure_lock_contention.py

```
@@ -66,16 +66,106 @@ async def worker(pool, playlist_id, bookmark_ids):
                 VALUES ($1, $2)
             """, values)
         except Exception as batch_err:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)